### PR TITLE
[MB-9027] Add DocumentViewerSidebar component to storybook

### PR DESCRIPTION
**Pull request by Ronolibert**
## Description

Adds generic document viewer sidebar component to storybook

## Setup
`make storybook`

## Code Review Verification Steps
* [ ] User facing changes have been reviewed by design.
* [ ] Request review from a member of a different team.
* [ ] Have the Jira acceptance criteria been met for this change?

## References

* [Jira story](https://dp3.atlassian.net/browse/MB-9027) for this change

## Screenshots

Screen Shot 2021-08-18 at 10 44 58 AM

### CODE REVIEW

**Review by pearl-truss — APPROVED** [review]
Code changes look good to me

**Review by YanZ777 — COMMENTED** [review]
Aside from my question, the changes look good to me.

File: src/pages/Office/DocumentViewerSidebar/DocumentViewerSidebar.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  description: string,
+  onClose: func.isRequired,
+};
+
+DocumentViewerSidebar.defaultProps = {
+  subtitle: '',
+  description: '',
+};
+
+DocumentViewerSidebar.Content = function Footer({ children }) {
```
[the remark this answers is not in the document]
Comment (Ronolibert) [reply src/pages/Office/DocumentViewerSidebar/DocumentViewerSidebar.jsx:39]: Nooo! Good catch, fixing this

**Review by YanZ777 — APPROVED** [review]
Changes look good!

**Review by cosburn — APPROVED** [review]
Looks good!

**Pull request merged**